.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: scala
+scala:
+- 2.10.4
+- 2.11.1
+jdk:
+- openjdk8
+script:
+- sbt test
+deploy:
+  skip_cleanup: true
+  provider: script
+  script: "./.travis/deploy.sh $TRAVIS_TAG"
+  on:
+    condition: '"${TRAVIS_SCALA_VERSION}" == "2.11.1"'
```

Comment:
Why do we keep `2.10.4` above if the condition is only met for `2.11.1` ?

Reply:
Because this only triggers deploy script, which runs `+publish`, so it publishes to all versions. This is mainly copy-pasted from `0.4.0`.